finetune/lora.py

```diff
@@ -112,7 +112,6 @@ def main(fabric: L.Fabric, data_dir: Path, checkpoint_dir: Path, out_dir: Path):
     fabric.print(f"Loading model {str(checkpoint_path)!r} with {config.__dict__}")
     with fabric.init_module(empty_init=False):
         model = GPT(config)
-        model.apply(model._init_weights)  # for the LoRA weights
```

Comment:
I agree with this removal. I encountered this also in #351, making the convergence poor. `_init_weights` was ever only meant for the llama model pretraining initialization (carried over from lit-llama) if I recall correctly.